language-support/hs/bindings/BUILD.bazel

```diff
@@ -72,7 +73,6 @@ da_haskell_test(
         ":quickstart.dar",
         "//ledger/sandbox:sandbox-binary",
     ],
-    flaky = True,  # FIXME Remove this once #1927 is solved
```

Comment:
Are you sure the flakiness was caused by this?

Reply:
nope. but I wanted to run on CI.
I will reinstate this line unless I think I have managed to fix it.

My gut feeling is that the`haskell-grpc` library does not mask async-exceptions in places it should, and the low-level c-code gets in some inconsistent state.

I plan to test this theory by some adding some aggressive masking in my code around certain calls to the `haskell-grpc` code.

Reply:
As discussed. We think the flakiness is due to some problem in `haskell-grpc`
Reinstated the`flaky = True` here.